src/phoebus_driver.cpp

```diff
@@ -524,13 +560,16 @@ TaskCollection PhoebusDriver::RungeKuttaStage(const int stage) {
   }
 
   // Fix up flux failures
+  // TODO (DO NOT MERGE) is this loop right
```

Comment:
Is this comment still relevant? Or just your comment below?

Reply:
Not relevant -- started going through refactoring fixup code, gave up for now. The comment survived